test/simple_socketio.test.js

```diff
@@ -456,7 +456,7 @@ describe("Tests using SimpleSocketIOClient", () => {
       // Reconnect should not be called yet
       expect(client.attemptReconnect).toHaveBeenCalledTimes(0);
     });
-    test("reconnect method exponentially increase delay for every attempt, stopping at the max value", () => {
```

Comment:
i think it would be helpful with a test that ensures that multiple calls to reconnect during the connection phase does not lead to multiple reconnection attempts in case of failure, just one.

Reply:
already implemented ⭐